app/views/_init-count.html

```diff
@@ -0,0 +1,11 @@
+<!--
+  Expects the following variables:
+  podTemplate
+ -->
+
+<span ng-if="pod.spec.initContainers.length" class="text-muted small">
+  <ng-pluralize count="pod.spec.initContainers.length"
+                when="{'1': '&nbsp;Init container','other': '&nbsp;{} Init containers'}">
```

Comment:
Let's always show a count, even if one like `1 Init Container`.

For case: either `<count> init containers` or `<count> Init Containers`? (The way you have it, `2 Init containers`, looks odd to me.)